app/integration/runtime/src/main/java/io/syndesis/integration/runtime/util/JsonSimplePredicate.java

```diff
@@ -165,4 +189,10 @@ static String toOgnl(final Matcher matcher) {
         return ognl.toString();
     }
 
+    private static boolean isCollectionPath(String expression) {
+        return COLLECTION_PATHS.stream()
```

Comment:
Perhaps leave `COLLECTION_PATHS` as `String` until we need more paths? It's a constant with only one element currently...

Reply:
I kinda like the idea to have everything in place to only add more collection paths in future. But this does not have to be a constant though. Pushed changes to this PR